extensions/BMO/lib/Reports/UserActivity.pm

```diff
@@ -322,6 +322,8 @@ sub report {
     $vars->{'incomplete_data'} = $incomplete_data;
     $vars->{'operations'}      = \@operations;
 
+    $vars->{user} = Bugzilla::User->new({ name => $input->{who}, cached => 1 });
```

Comment:
This is inside the for loop that stores each activity item. So depending on how many activity rows there are, it will be set each time. Also user_activity.html allows for displaying more than one person grouped together. Normally from user_profile.html, the link to user_activity.html would only be a single person so wouldn't be an issue with that case. But we need to be able to handle both cases.

So out side of the for loop you need to create $var->{users}:

```perl
$var->{users}
  = [ map { Bugzilla::User->new({ name => $_, cache => 1}) } @who ];
```